transport-classes-io_uring/src/main/java/io/netty/channel/uring/AbstractIoUringStreamChannel.java

```diff
@@ -281,10 +314,17 @@ protected int scheduleWriteSingle(Object msg) {
                 ops = fileRegion.splice(fd);
             } else {
                 ByteBuf buf = (ByteBuf) msg;
-                ops = IoUringIoOps.newWrite(fd, (byte) 0, 0,
-                        IoUring.memoryAddress(buf) + buf.readerIndex(), buf.readableBytes(), nextOpsId());
+                long address = IoUring.memoryAddress(buf) + buf.readerIndex();
+                int length = buf.readableBytes();
+                short opsid = nextOpsId();
+                if (IoUring.isSendZcSupported() &&
```

Comment:
IoUringDomainSocketChannel does not support sendzc/sendmsgZc. In my tests, it throws io.netty.channel.unix.Errors$NativeIoException: io_uring sendzc(..) failed Operation not supported.
Should we consider disabling the sendzc feature for IoUringDomainSocketChannel?

Reply:
Yeah let me take care of it